configurator/src/main.rs

```diff
@@ -801,201 +798,226 @@ fn main() -> Result<(), anyhow::Error> {
-                            "✅".to_owned()
-                        } else {
-                            "❌".to_owned()
-                        },
-                        description: Some("Until this value is ✅, you may not be able to see transactions sent to your on chain wallet.".to_owned()),
-                        copyable: false,
-                        qr: false,
-                        masked: false,
-                    },
-                    synced_to_graph: Property {
-                        value_type: "string".to_owned(),
-                        value: if ni.synced_to_graph {
-                            "✅".to_owned()
-                        } else {
-                            "❌".to_owned()
-                        },
-                        description: Some("Until this value is ✅, you will experience problems sending payments over lightning.".to_owned()),
-                        copyable: false,
-                        qr: false,
-                        masked: false,
-                    },
-                    lnd_connect_grpc,
-                    lnd_connect_rest,
-                    node_uri,
-                    alias: Property {
-                        value_type: "string".to_owned(),
-                        value: alias,
-                        description: Some("The human readable name your node can identify as on the Lightning Network".to_owned()),
-                        copyable: false,
-                        qr: false,
-                        masked: false,
-                    }
-                },
-            };
-            serde_yaml::to_writer(File::create("/root/.lnd/start9/stats.yaml")?, &stats)?;
-            stats
-        }
-        None => {
-            const PROPERTIES_FALLBACK_MESSAGE: &str =
-                "Could not find properties. The service might still be starting";
-            let stats_path = Path::new("/root/.lnd").join("start9/stats.yaml");
-            if stats_path.exists() {
-                let stats: Properties =
-                    serde_yaml::from_reader(File::open(stats_path).unwrap()).unwrap();
-                stats
-            } else {
-                let stats = Properties {
-                    version: 2,
-                    data: Data::NotReady {
-                        not_ready: Property {
-                            value_type: "string".to_owned(),
-                            value: PROPERTIES_FALLBACK_MESSAGE.to_owned(),
-                            description: Some(
-                                "Fallback Message When Properties could not be found".to_owned(),
-                            ),
-                            copyable: false,
-                            qr: false,
-                            masked: false,
-                        },
-                    },
-                };
-                stats
-            }
-        }
-    };
-    Ok(stats)
-}
+// fn get_stats(
```

Comment:
remove all commented out code unless its needed. it can always be referenced in the git history